frontend/tests/features/collection/collection.test.ts

```diff
@@ -35,13 +38,13 @@ type CollectionFormInput = Pick<
 
 describe("Collection", () => {
   describeIfDeployed("Logged In Tests", () => {
-    it("creates and deletes a collection", async () => {
+    test("creates and deletes a collection", async ({ page }) => {
```

Comment:
Playwright test runner doesn't have global `page` object, so we need to access `page` explicitly through callback function argument

here and throughout test files